Makefile

```
@@ -551,4 +560,34 @@ ios-appstore-verify: ## Verify App Store submission readiness (repo-local, no up
 	$(VENV_PYTHON) -m pytest -q tests/test_release_reviewer_packet_hashes.py
 	@echo "$(GREEN)App Store submission readiness verified$(NC)"
 
-.PHONY: all help venv venv-sync setup-automation dev test test-fast validate-min validate-changed cov cov-check cov-html lint fmt fmt-check security pre-commit quick-check auto-push safe-push feature sync-main status clean check-all fix-all ci smoke-auto smoke-8000 smoke-8001 docker-build docker-build-dev docker-run docker-run-dev docker-stop docker-clean docker-logs docker-shell bandit-full diff-cov typecheck verify verify-env openapi frontend-install openapi-check ios-test ios-snapshot ios-appstore-validate ios-appstore-upload ios-appstore-upload-privacy ios-appstore-verify icon-silhouette-lock icon-silhouette-check icon-core-validate design-guard tokens-build tokens-check design-validate design-execute design-verify design-list
+# --- Dev Container targets ---------------------------------------------------
+
+devcontainer-bootstrap: ensure-python-proxy ## Install deps + hooks inside dev container
```

checkmake (0.3.2)

[warning] Target body for "devcontainer-bootstrap" exceeds allowed length of 5 lines (13).

(maxbodylength)